README.md

```diff
@@ -1,2 +1,30 @@
 # AWS-IPFS-Infrastructure
```

Comment:
Tip - https://terraform-docs.io can be used to generate some additional information here

Reply:
Cool tip. I was able to create module specific documentation using that.
- [api-gateway-to-s3](https://github.com/web3-storage/AWS-IPFS-Infrastructure/blob/api-gateway-to-s3/terraform/modules/api-gateway-to-s3/api-gateway-to-s3.md)
- [indexing-subsystem](https://github.com/web3-storage/AWS-IPFS-Infrastructure/blob/api-gateway-to-s3/terraform/modules/indexing-subsystem/indexing-subsystem.md)

Reply:
Whenever we have a pipeline for this project, we could integrate this tool for continuous documentation update :smile: